### ISSUE TITLE
3.0.9 - Test failure on Debian 10

### ISSUE THREAD

**Issue by Hopman** [libical/libical#487]
Building on Debian 10 (buster):
```
% libical-3.0.9/build  uname -a
Linux sharp-spider 4.19.0-16-amd64 #1 SMP Debian 4.19.181-1 (2021-03-19) x86_64 GNU/Linux
```
```
% mkdir build
% cd build
% cmake ..
% make
% make test
```
1 test fails:
```
39/44 Test #39: testvcal .........................   Passed    0.00 sec
      Start 40: process
40/44 Test #40: process ..........................   Passed    0.00 sec
      Start 41: timezones
41/44 Test #41: timezones ........................***Failed    0.54 sec
      Start 42: builtin_timezones
42/44 Test #42: builtin_timezones ................   Passed    6.35 sec
      Start 43: icalrecurtest-r
43/44 Test #43: icalrecurtest-r ..................   Passed    0.11 sec
      Start 44: icalrecurtest
44/44 Test #44: icalrecurtest ....................   Passed    0.02 sec

98% tests passed, 1 tests failed out of 44

Total Test time (real) =   7.44 sec

The following tests FAILED:
	 41 - timezones (Failed)
Errors while running CTest
make: *** [Makefile:86: test] Error 8
```
The verbose output shows the failures, one example:
```
America/Whitehorse: day 000: okay again: 2021-01-01 19:00:00 UTC = libc 2021-01-01 12:00:00 dst 0
Indian/Cocos: day 000: first failed: 2021-01-01 05:30:00 UTC = libc 2021-01-01 12:00:00 dst 0 != libical 2021-01-01 11:57:40 dst 1
BEGIN:VTIMEZONE
TZID:/softwarestudio.org/Indian/Cocos
X-LIC-LOCATION:Indian/Cocos
BEGIN:STANDARD
TZNAME:+0630
DTSTART:20360204T062816
TZOFFSETFROM:+062740
TZOFFSETTO:+0630
RRULE:FREQ=YEARLY;UNTIL=20360206T235816Z;BYDAY=1TH;BYMONTH=2
END:STANDARD
BEGIN:STANDARD
TZNAME:+0630
DTSTART:20380119T094407
TZOFFSETFROM:+0630
TZOFFSETTO:+0630
RRULE:FREQ=YEARLY;BYDAY=3TU;BYMONTH=1
END:STANDARD
END:VTIMEZONE

```
Please let me know if you need any more details/information

**Comment by ksmurchison** [comment]
Do you have the ability to test the yet-to-be-released 3.0.10?  I believe this will solve the problem.

https://github.com/libical/libical/tree/3.0

**Comment by Hopman** [comment]
I tested the 3.0 branch, and the tests all succeed, so the problem seems fixed in 3.0.10 as you mentioned.

Thanks

**Comment by plaimbock** [comment]
I also saw test 41 fail with 3.0.9 on CentOS 8.3 with gcc9. Can confirm it's fixed in git rev 2d02789575f09411c5ba49ed2c80221df10e5c37

**Comment by Hopman** [comment]
Fixed in next version
